rust/src/config.rs

```diff
@@ -28,8 +40,10 @@ pub struct FeaturesConfig {
 /// Please see the book for more details.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SettingsConfig {
+    /// Time to sleep between tests.
```

Comment:
This line isn't accurate.  `naptime` isn't used between tests; it's used within tests.  The requirement is that it should be greater than the timestamp granularity of the file system under test.

Reply:
Fixed